ui/src/dashboards/actions/index.js

```diff
@@ -122,25 +135,34 @@ export const editTemplate = (dashboardID, templateID, updates) => ({
 
 // Stub Template Variables Data
 
-const templates = [
+const tempVars = [
   {
-    id: 1,
-    type: 'measurement',
+    id: '1',
+    type: 'query',
     label: 'test query',
-    code: '$HOSTS',
+    tempVar: '$REGION',
     query: {
       db: 'db1',
+      rp: 'rp1',
       measurement: 'm1',
       text: 'SHOW TAGS WHERE HUNTER = "coo"',
```

Comment:
Yes and that is pretty lame ... would love to see it change to what it really is ... `influxql`